com.github.halfmexican.Mingle.metainfo.xml.in

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.github.halfmexican.Mingle.desktop</id>
```

Comment:
id use the reversed dns names NEVER ends in `.desktop`

Reply:
this must be fixed upstream when you fix the app-id.

Reply:
fixed!